_posts/2018-04-09-synching-sia.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Syncing Sia in 73 minutes
+layout: post
+date: "2018-04-09"
+description: "An experiment to see how fast I can perform the initial Sia blockchain sync"
+tags:
+- sia
+- blockchain
+- benchmark
+permalink: "/sia-sync-73-min/"
+---
+
+*This is a guest post by Sia community contributor [Thomas Bennett](https://github.com/tbenz9).*
+
+I can sync Sia from a block height of 0 to 149000 in 73 minutes.  Here’s how I do it.
+
+## Zero subscribers
```

Comment:
Can we start with an overview of the steps involved before we dive into each one? Just like a bulleted list saying each thing we changed.